dirsrvtests/tests/suites/clu/dsidm_account_test.py

```diff
@@ -0,0 +1,148 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2020 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+import time
+import subprocess
+import pytest
+import logging
+import os
+
+from lib389 import DEFAULT_SUFFIX
+from lib389.cli_idm.account import list, get_dn, lock, unlock, delete, modify, rename, entry_status, \
+    subtree_status, reset_password, change_password
+from lib389.topologies import topology_st
+from lib389.cli_base import FakeArgs
+from lib389.utils import ds_is_older
+from lib389.idm.user import nsUserAccounts
+
+pytestmark = pytest.mark.tier0
+
+LOG_FILE = '/tmp/dsidm_account.log'
```

Comment:
I'm a bit worried about this use of a log file. Can't python logging log into a in memory buffer?

Reply:
Actually, since your using FakeArgs and the cli test framework, there is a log capturing type. See /src/lib389/lib389/tests/cli and how it captures the output and tests it.

Reply:
Thanks for reminding me about that, I changed the log capture accordingly. Please review.